modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -202,7 +197,7 @@ $ oc apply -f ingress-autoscaler.yaml
 +
 [source,terminal]
 ----
-$ oc get ingresscontroller/default -o yaml | grep replicas:
+$ oc get -n openshift-ingress-operator ingresscontroller/default -o yaml | grep replicas
```

Comment:
```suggestion
$ oc get -n openshift-ingress-operator ingresscontroller/default -o yaml | grep replicas:
```

This `:` was intentional I believe. If you don't have it, you'll get extra unnecessary info:
```
$ oc get -n openshift-ingress-operator ingresscontroller/default -o yaml | grep replicas
  replicas: 2
    message: Minimum replicas requirement is met
    message: All replicas are available
```
Vs:
```
$ oc get -n openshift-ingress-operator ingresscontroller/default -o yaml | grep replicas:
  replicas: 2
```